libpod/container_internal_linux.go

```diff
@@ -377,6 +377,15 @@ func (c *Container) setupSystemd(mounts []spec.Mount, g generate.Generator) erro
 		}
 		g.AddMount(systemdMnt)
 	}
+	if rootless.IsRootless() {
```

Comment:
How is this writable?

Reply:
Is this safe? I know systemd really dislike having to share CGroups with anyone else without explicit units and delegation

Reply:
I think systemd detects in what user slice it is running and uses that.  If I specify "ro", I get from systemd in the container:

```
Failed to create /user.slice/user-1000.slice/user@1000.service/gnome-terminal-server.service/7a24e4a13856680aebdb0f05657eea52ac353196e95bde23461066f665f5df98/init.scope control group: Read-only file system
```

Reply:
I think this is safe enough.  
@giuseppe if you enter the container and go down the /sys/fs/cgroup/systemd/libpod-uuild...
Are the files owned by -1 or by the UserNS Root?

Reply:
they are owned by root in the user namespace:

```
[root@be714e845c11 /]# ls -l /sys/fs/cgroup/systemd/user.slice/user-1000.slice/user@1000.service/gnome-terminal-server.service/be714e845c116204fd737d52aa6b9989786bf37984dec99b74717d469ece5c80/init.scope
total 0
-rw-r--r--. 1 root root 0 Nov  5 23:38 cgroup.clone_children
-rw-r--r--. 1 root root 0 Nov  5 23:38 cgroup.procs
-rw-r--r--. 1 root root 0 Nov  5 23:38 notify_on_release
-rw-r--r--. 1 root root 0 Nov  5 23:38 tasks
```

Reply:
Awesome.